### PR TITLE
chore(websocket): golang.org/x/net/websocket -> github.com/gorilla/websocket

### LINKED ISSUE #219: Replace the use of golang.org/x/net/websocket
As specified in this dependency [documentation](https://pkg.go.dev/golang.org/x/net@v0.0.0-20210405180319-a5a99cb37ef4/websocket?utm_source=gopls):
> This package currently lacks some features found in alternative and more actively maintained WebSocket packages:
>
> https://godoc.org/github.com/gorilla/websocket
> https://godoc.org/nhooyr.io/websocket

github.com/gorilla/websocket is the one we already use on some prjects, let's use this one

### PR DESCRIPTION
**Pull request by QuentinEscudierScalingo**
Replace the use of golang.org/x/net/websocket, now go-scalingo is using github.com/gorilla/websocket

Fix #219

### CODE REVIEW

File: deployments.go [hunk 1]
```diff
@@ -190,7 +190,16 @@ func (c *Client) DeploymentStream(ctx context.Context, deployURL string) (*webso
 	if err != nil {
 		return nil, errgo.Notef(err, "fail to generate token")
 	}
-	authString, err := json.Marshal(&AuthStruct{
+
+	header := make(http.Header)
```
Comment (EtienneM) [inline deployments.go:194]: convention: We rarely use `make` in our codebase. 
```suggestion
	header := http.Header{}
```

File: deployments.go [hunk 2]
```diff
@@ -190,7 +190,16 @@ func (c *Client) DeploymentStream(ctx context.Context, deployURL string) (*webso
...
+	header.Add("Origin", "http://scalingo-cli.local/1")
+	conn, resp, err := websocket.DefaultDialer.DialContext(ctx, deployURL, header)
+	if err != nil {
+		return nil, errgo.Mask(err, errgo.Any)
```
Comment (EtienneM) [inline deployments.go:198]: convention: We don't use `Mask` anymore. Please use `Notef`.

File: mocks.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "base_directory": "github.com/Scalingo/go-scalingo",
-  "base_package": "github.com/Scalingo/go-scalingo/v5",
+  "base_package": "github.com/Scalingo/go-scalingo/v6",
```
Comment (EtienneM) [inline mocks.json:3]: I'm surprised this file changes but not `mocks_sig.json`. Are you sure you didn't forgot to commit it?

File: go.mod [hunk 4]
```diff
@@ -12,6 +12,7 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/gorilla/websocket v1.5.0
```
Comment (EtienneM) [inline go.mod:15]: Did you correctly vendored your dependencies? This should be in the previous `require` block as it a direct dependency.

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: mocks_sig.json [hunk 5]
```diff
@@ -1,35 +1,35 @@
...
-  "github.com/Scalingo/go-scalingo.RegionMigrationsService": "79 37 8f 49 6a 2f 79 d7 91 a7 6b 5c 8a 8f 66 54 5d 32 0f 5d",
-  "github.com/Scalingo/go-scalingo.RegionsService": "bc d0 b0 22 5b 7c df 31 02 b7 b3 4d 1f b4 9a 88 a2 7c 8a 6a",
-  "github.com/Scalingo/go-scalingo.RunsService": "b3 27 40 57 21 c2 64 f8 08 89 c9 0e 06 45 0d 1b ff 5b 5a 45",
-  "github.com/Scalingo/go-scalingo.SCMRepoLinkService": "bd d9 5a a3 19 18 70 9e 5c 0e 18 a3 76 ea 6e 9d d1 19 18 41",
-  "github.com/Scalingo/go-scalingo.SignUpService": "ae 46 80 88 8e b6 bd c3 ea d9 2c e5 d4 b5 37 ac a9 e8 4f cf",
-  "github.com/Scalingo/go-scalingo.SourcesService": "05 1f 06 73 0a 43 2a 6a 35 d2 b9 41 d2 be 8e 7c ad b0 63 3e",
-  "github.com/Scalingo/go-scalingo.TokensService": "32 2d 05 42 b4 9c a9 3b 1b 6c ce 0e da 18 cf 6f 8f cb fc 2a",
-  "github.com/Scalingo/go-scalingo.UsersService": "9e c5 75 1e aa 05 dd a7 a3 17 5f 4e 82 80 e2 34 55 a2 70 df",
-  "github.com/Scalingo/go-scalingo.VariablesService": "77 27 0b 8a 6b ac c9 c8 c3 b0 e0 f8 cf 7d 6e a0 63 9d 95 03",
+  "github.com/Scalingo/go-scalingo.API": "NOFILE",
```
Comment (EtienneM) [inline mocks_sig.json:2]: Please call `gomock_generator` a second time. This is due to a known bug in `gomock_generator`.

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: mocks_sig.json (hunk 5 above)
Comment (QuentinEscudierScalingo) [reply mocks_sig.json]: Sorry, I didn't see that

**Review by EtienneM — APPROVED** [review]

**Pull request merged**